### PR TITLE
[VL] Upgrade celeborn CI to 0.3.0

### PR DESCRIPTION
**Pull request by kerwin-zk**
## What changes were proposed in this pull request?
Upgrade celeborn CI to 0.3.0

### CODE REVIEW

**Comment by kerwin-zk** [comment]
@pan3793

**Review by pan3793 — APPROVED** [review]

File: tools/gluten-it/common/src/main/scala/io/glutenproject/integration/tpc/Constants.scala [hunk 1]
```diff
@@ -43,6 +43,8 @@ object Constants {
     .set("spark.sql.parquet.enableVectorizedReader", "true")
     .set("spark.plugins", "io.glutenproject.GlutenPlugin")
     .set("spark.shuffle.manager", "org.apache.spark.shuffle.gluten.celeborn.CelebornShuffleManager")
+    .set("spark.driver.extraClassPath", "/opt/hadoop/*")
+    .set("spark.executor.extraClassPath", "/opt/hadoop/*")
```
Comment (zhztheplayer) [inline tools/gluten-it/common/src/main/scala/io/glutenproject/integration/tpc/Constants.scala:47]: Can we insteadly add gluten-it options, for example, `--extra-conf=spark.driver.extraClassPath=/opt/hadoop/*` to velox_be.yml?

Comment (kerwin-zk) [reply tools/gluten-it/common/src/main/scala/io/glutenproject/integration/tpc/Constants.scala]: @zhztheplayer done.

**Comment by PHILO-HE** [comment]
Please do a code rebase to fix the CI failure. Thanks!

**Review by ulysses-you — APPROVED** [review]

**Comment by ulysses-you** [comment]
@PHILO-HE @zhztheplayer can we get this in ?

**Comment by zhztheplayer** [comment]
Thanks!

**Pull request merged**